spi/core-spi/src/main/java/org/eclipse/dataspaceconnector/spi/types/domain/transfer/DataFlowRequest.java

```diff
@@ -85,12 +89,31 @@ public Map<String, String> getProperties() {
         return properties;
     }
 
+    /**
+     * Trace context for this request
+     */
+    @Override
+    public Map<String, String> getTraceContext() {
+        return traceContext;
+    }
+
+    /**
+     * A builder initialized with the current DataFlowRequest
+     */
+    public Builder toBuilder() {
```

Comment:
is there any significant advantage of this over a  setter in the `DataFlowRequest`?

Reply:
The advantage is to keep `DataFlowRequest` immutable